meteor/server/api/ingest/packageInfo.ts

```diff
@@ -0,0 +1,161 @@
+import { ExpectedPackageDBType, ExpectedPackageId, ExpectedPackages } from '../../../lib/collections/ExpectedPackages'
+import { MediaObject, MediaObjId } from '../../../lib/collections/MediaObjects'
+import { PackageInfoDB } from '../../../lib/collections/PackageInfos'
+import { RundownId, Rundowns } from '../../../lib/collections/Rundowns'
+import { SegmentId } from '../../../lib/collections/Segments'
+import { assertNever, lazyIgnore } from '../../../lib/lib'
+import { logger } from '../../logging'
+import { runIngestOperationWithCache } from './lockFunction'
+import { Meteor } from 'meteor/meteor'
+import { regenerateSegmentsFromIngestData } from './generation'
+
+/** to-be @deprecated hack for backwards-compatibility*/
```

Comment:
In or out? TODO or to-deprecate? Which is it?

Reply:
Julian and I decided to remove this method for #525 ... see commit https://github.com/nrkno/tv-automation-server-core/pull/525/commits/0301a5e987ff6d8836e9f0aeb04c61834582b37b